LouisBarranqueiro/reapop
# fix: notificationButtons style not passed to the buttons container

### Related issues: #452
### Changes
- Passed the `notificationButtons` styles from the theme to the buttons container.
- Added to the tests to reflect the change.

### Testing guide
- Run `npm run test`.

Eventually : 
- Create a custom theme as described in the inline CSS [guide](https://github.com/LouisBarranqueiro/reapop/blob/main/src/themes/types.ts).
- Pass custom styles to the buttons container via notificationButtons.
- See that the styles are rendered (inline styles are not created for the buttons container).

### Checklist
- [x] I ran the tests with `npm run test` 
- [x] I added tests for the bug I fixed (if applicable)
- [ ] I updated the README or the DOCUMENTATION (if applicable)
- [ ] I updated the demo (if applicable)
- [x] I verified my changes on all browsers listed below:
  - [ x] Firefox


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/louisbarranqueiro/reapop/453)
